tests/strictequal.test.js

```diff
@@ -0,0 +1,52 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictEqual = require('../strictequal')
+
+describe('test 1 fail on parameter types types', () => {
+  describe('test 1 parameter are different types', () => {
```

Comment:
Remember that `describe` provides structure, you wouldn't have nested `describe`s for each test like this. Having a `describe` for the entire module, within that a `describe` for each function in the module, and then in that `it`s for each test is the preferred structure. This kind of breakdown makes it hard to read the code and the results.